lighthouse-core/audits/works-offline.js

```diff
@@ -27,6 +27,7 @@ class WorksOffline extends Audit {
       category: 'Offline',
       name: 'works-offline',
       description: 'URL responds with a 200 when offline',
+      helpText: 'If your app is supposed to work offline and you\'re failing this test, then something is wrong with your offline implementation. <a href="https://developers.google.com/web/tools/lighthouse/audits/http-200-when-offline" target="lighthouse" rel="noopener noreferrer">Learn more</a>.',
```

Comment:
I think we can be more constructive :) How about mentioning that a quality PWA should work offline...